metricbeat/module/rabbitmq/exchange/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Any chance you could create a data.json here? The command you need is as following:
```
go test -tags=integration github.com/elastic/beats/metricbeat/module/rabbitmq/exchange/... -data
```

It assumes you have rabbitmq running on localhost.

Reply:
Will do.

Reply:
This command does not generate the full json file (the message stats are missing). Am I missing something?

```
{
    "@timestamp": "2017-10-12T08:05:34.853Z",
    "beat": {
        "hostname": "host.example.com",
        "name": "host.example.com"
    },
    "metricset": {
        "host": "localhost:15672",
        "module": "rabbitmq",
        "name": "exchange",
        "rtt": 115
    },
    "rabbitmq": {
        "exchange": {
            "arguments": {},
            "auto_delete": false,
            "durable": true,
            "internal": false,
            "name": "",
            "type": "direct",
            "vhost": "/"
        }
    }
}
```